resource/interface_factories.cpp.em

```diff
@@ -98,6 +108,43 @@ get_service_factory_@(ros2_package_name)__@(interface_type)__@(interface.message
   return nullptr;
 }
 @
+
+std::unique_ptr<ActionFactoryInterface>
+get_action_factory_@(ros2_package_name)__@(interface_type)__@(interface.message_name)(const std::string & ros_id, const std::string & package_name, const std::string & action_name)
+{
+@[if not mapped_actions]@
+  (void)ros_id;
+  (void)package_name;
+  (void)action_name;
+@[end if]@
+@[for action in mapped_actions]@
+  if (
+    ros_id == "ros1" &&
+    package_name == "@(action["ros1_package"])" &&
+    action_name == "@(action["ros1_name"])"
+  ) {
+    return std::unique_ptr<ActionFactoryInterface>(new ActionFactory_2_1<
+      @(action["ros1_package"])::@(action["ros1_name"])Action,
+      @(action["ros2_package"])::action::@(action["ros2_name"])
+    >);
+  }
+  else if
+    (
+      ros_id == "ros2" &&
+      package_name == "@(action["ros2_package"])" &&
+      action_name == "action/@(action["ros2_name"])"
+  ) {
+    return std::unique_ptr<ActionFactoryInterface>(new ActionFactory_1_2<
+      @(action["ros1_package"])::@(action["ros1_name"])Action,
+      @(action["ros2_package"])::action::@(action["ros2_name"])
+    >);
+  }
+@[end for]@
+  return nullptr;
+}
+@
+}
+@
```

Comment:
```suggestion
```

This was closing the ros1_bridge namespace and causing errors on the service creation.